test/test_jit.py

```diff
@@ -15618,6 +15618,11 @@ def add_autograd_test(
     if 'complex' in variant_name or name in ['view_as_complex', 'complex']:
         return
 
+    # Disable tests for lu from common_methods_invocations.py
+    # TODO(@nikitaved) Envable jit tests once autograd.Function does support scripting
+    if 'square' in variant_name and name in ['lu']:
+        return
+
```

Comment:
@albanD, I personally do not like these lines, is there a better way to disable tests for AutogradJIT?

Reply:
nit in the todo: "enable"

No I don't know how we can do that from looking at test_jit. That's why I didn't say anything.
Let me check if someone has an idea.

Reply:
From @eellison, you can add it there: https://github.com/pytorch/pytorch/blob/161490d441ba70ad7f32e1d373d21fdb0dc26a81/torch/testing/_internal/jit_metaprogramming_utils.py#L324
Does that help?

Reply:
Nope, does not work for me. The tests show up and they fail. Surprisingly, the list contains `test_norm_fro`, for example, and test_jit does generate a suite for it, although it passes.
The failure message for my tests is:
```
E           RuntimeError:                                                                                                                         
E           Could not export Python function call '_LU'. Remove calls to Python functions before export. Did you forget to add @script or @script_
method annotation? If this is a nn.ModuleList, add it to __constants__:        
```
So, I guess, we just leave things as they are now?

Reply:
Elias suggest this one maybe: https://github.com/pytorch/pytorch/blob/7a5924f010dd523da1a9ae74e2db61a91a53f80e/test/test_jit.py#L15421

Reply:
Same ;(